projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -62,12 +60,15 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
             >
             </ng-container>
             <li class="accordion-list-item row sub-title text-break">
-              <div class="col-4">
-                {{ 'quote.selectQuoteNo' | cxTranslate }}
-              </div>
-              <div *ngFor="let cart of quotes?.carts" class="col-4">
+              <div
+                *ngFor="let cart of quotes?.carts; index as i"
+                class="col"
+                [ngClass]="
+                  i === 0 ? 'offset-' + 12 / (1 + quotes?.carts.length) : ''
```

Comment:
This seems a bit over-engineered. Can we perhaps simplify this?

Reply:
As we agreed, this will stay as it is, because of the need to dynamically create `offset` class for both cases 2 or 3 quotes.